specification.md

```diff
@@ -3796,29 +3776,59 @@ You can combine retry and onError definitions to define powerful error handling
 
 ```json
 {
-"retry": [
+"retries": [
   {
-    "expression": "{{ $.errors[?(@.name == 'FunctionExecutionError')] }}",
-    "maxAttempts": 3,
-    "interval": "PT2M"
-  },
-  {
-    "expression": "{{ $.errors[?(@.name != 'FunctionExecutionError')] }}",
-    "maxAttempts": 2,
-    "interval": "PT1M"
-  }
-],
-"onError": [
-  {
-    "expression": "{{ $.errors[?(@.name == 'FunctionExecutionError')] }}'",
+    "name": "Service Call Timeout Retry Strategy",
+    "delay": "PT1M",
+    "maxAttempts": 4,
     "transition": {
```

Comment:
This is a typo to have `transition` defined here, right?

Reply:
yep :)